cipher/aes.go

```diff
@@ -0,0 +1,68 @@
+package cipher
+
+import (
+	"context"
+
+	"github.com/gtank/cryptopasta"
+
+	"github.com/ory/herodot"
+
+	"github.com/pkg/errors"
+
+	"github.com/ory/kratos/driver/config"
+)
+
+type AES struct {
+	c AESConfiguration
+}
+
+type AESConfiguration interface {
+	config.Provider
+}
+
+func NewCryptAES(c AESConfiguration) *AES {
+	return &AES{c: c}
+}
+
+// Encrypt return a AES encrypt of plaintext
+func (a *AES) Encrypt(ctx context.Context, clearString string) ([]byte, error) {
+	if len(clearString) == 0 {
+		// do nothing if empty instead of return an error
+		//return nil, errors.WithStack(herodot.ErrInternalServerError.WithReason("Can not encrypt empty string."))
```

Comment:
I had to comment that because the GitHub Oauth2 does not return the refresh_token.
Don't know. I'll try with a new package version of GitHub-go

Reply:
it's not an issue
It's the normal GitHub oauth2 behaviour

by the way the refresh token is present with GitHub apps